tests/scripts/build-release.sh

```diff
@@ -0,0 +1,39 @@
+/bin/bash -e
+set -ex
+
+#############
+# VARIABLES #
+#############
+BRANCH_NAME=$1
+
+#############
+# FUNCTIONS #
+#############
+
+function  master_branch() {
+    build/run make -j$(nproc) VERSION=0 build.all
```

Comment:
If this step fails, it aborts the rest of the script, right? Of course we can't push if the build fails

Reply:
Why VERSION=0?

Reply:
As mentioned earlier we need to fix this before pushing this. We use `VERSION=0` because the helm build was breaking. Now we need to fix this instead of the workaround.